app/(main)/markets/MarketDetails.tsx

```diff
@@ -81,8 +83,18 @@ export const MarketDetails = ({ id }: MarketDetailsProps) => {
                   </Tag>
                 </a>
                 {marketModel.isClosed ? (
-                  <Tag className="w-fit capitalize" size="sm" colorScheme="quaternary">
-                    Market Closed
+                  <Tag
+                    className="w-fit gap-1 capitalize"
+                    colorScheme="quaternary"
+                    size="sm"
+                  >
+                    <p>Market Closed: Outcome</p>
```

Comment:
let's simplify, it does not need "outcome" text we can use a html title with "the outcome for this market is NO" 

```suggestion
                    <p>Market Closed</p>
```